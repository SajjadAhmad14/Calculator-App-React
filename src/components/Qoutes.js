import React from 'react';

const Qoutes = () => (
  <div>
    <h1>This is qoutes page</h1>
  </div>
);

export default Qoutes;
