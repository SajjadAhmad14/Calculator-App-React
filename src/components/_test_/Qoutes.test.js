import React from 'react-dom';
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react';
import Qoutes from '../Qoutes';

describe('Test Qoutes component', () => {
  it('Add screenshot test', () => {
    const { asFragment } = render(<Qoutes />);
    expect(asFragment(<Qoutes />)).toMatchSnapshot();
  })
});