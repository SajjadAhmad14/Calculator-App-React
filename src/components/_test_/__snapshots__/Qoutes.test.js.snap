// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Qoutes component Add screenshot test 1`] = `
<DocumentFragment>
  <div
    class="qoutesContainer"
  >
    <p
      class="qoutes"
    >
      If everything seems under control, youâ€™re just not going fast enough.
    </p>
  </div>
</DocumentFragment>
`;
